content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md

```diff
@@ -0,0 +1,206 @@
+published in late 2020.**
+
+This document goes over some frequently asked questions regarding the
+deprecation and removal of _dockershim_, that was
+[announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/)
+as a part of the Kubernetes v1.20 release. For more detail
+on what that means, check out the blog post
+[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/).
+
+Also, you can read [check whether dockershim removal affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
+to determine how much impact the removal of dockershim would have for you
+or for your organization.
+
+As the Kubernetes 1.24 release has become imminent, we've been working hard to try to make this a smooth transition.
+
+- We've written a blog post detailing our [commitment and next steps](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/).
+- We believe there are no major blockers to migration to [other container runtimes](/docs/setup/production-environment/container-runtimes/#container-runtimes).
+- There is also a [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) guide available.
+- We've also created a page to list
+  [articles on dockershim removal and on using CRI-compatible runtimes](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/).
+  That list includes some of the already mentioned docs, and also covers selected external sources
+  (including vendor guides).
+
+### Why is the dockershim being removed from Kubernetes?
+
+Early versions of Kubernetes only worked with a specific container runtime:
+Docker Engine. Later, Kubernetes added support for working with other container runtimes.
+The CRI standard was [created](/blog/2016/12/container-runtime-interface-cri-in-kubernetes/) to
+enable interoperability between orchestrators (like Kubernetes) and many different container
+runtimes.
+Docker Engine doesn't implement that interface (CRI), so the Kubernetes project created
+special code to help with the transition, and made that _dockershim_ code part of Kubernetes
+itself.
+
+The dockershim code was always intended to be a temporary solution (hence the name: shim).
+You can read more about the community discussion and planning in the
+[Dockershim Removal Kubernetes Enhancement Proposal][drkep].
+In fact, maintaining dockershim had become a heavy burden on the Kubernetes maintainers.
+
+Additionally, features that were largely incompatible with the dockershim, such
+as cgroups v2 and user namespaces are being implemented in these newer CRI
+runtimes. Removing support for the dockershim will allow further development in
+those areas.
+
+[drkep]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2221-remove-dockershim
+
+### Can I still use Docker Engine in Kubernetes 1.23?
+
+Yes, the only thing changed in 1.20 is a single warning log printed at [kubelet]
```

Comment:
Title question mentions 1.23 and answer states only 1.20
Maybe it can read “only thing changed since version 1.20 up to 1.23 is a...”